app/controllers/Application.scala

```diff
@@ -75,7 +75,14 @@ object Application extends Controller with PanDomainAuthActions {
       sections <-  getSortedSections
       desks <- getSortedDesks
       sectionsInDesks <- getSectionsInDesks
-      commissioningDesks <- TagService.getTags(Config.tagManagerUrl+ "/hyper/tags?limit=100&query=tracking/commissioningdesk/&type=tracking&searchField=path").map(_.getOrElse(List[Tag]()))
+      commissioningDesks <- TagService.getTags(Config.tagManagerUrl+
+        "/hyper/tags?limit=100&query=tracking/commissioningdesk/&type=tracking&searchField=path")
+        .map(_.getOrElse(List[Tag]())) recoverWith {
+          case e: Exception => {
+            Logger.error(s"error in fetching tags: $e")
```

Comment:
When logging exceptions, I think it's better to do Logger.error("error fetching tags: ${e.getMessage}", e) - that way the entire exception gets passed to the logger and can be handled nicely, rather than the full string representation with stack tract etc. being printed in the message.